source/index.txt

```diff
@@ -23,91 +23,105 @@ MongoDB Node Driver
-
-Developer Hub
-~~~~~~~~~~~~~
-
-The Developer Hub contains content, resources, community, and social
-engagement for developers.
-
-To view articles on Node.js, see the `How To's and Articles page
-<https://developer.mongodb.com/learn/?content=Articles&text=Node.js>`_.
-
-To view the forum on Node.js, see the `forums page
-<https://developer.mongodb.com/community/forums/tag/node-js>`_.
-
-MongoDB University
-~~~~~~~~~~~~~~~~~~
-
-MongoDB University contains free courses to inspire and educate everyone
-to learn MongoDB. It has the best-in-class training and empowers
-everyone to be successful.
+Welcome to the MongoDB Node.js driver documentation. 
 
-Take the free online course taught by MongoDB
-`````````````````````````````````````````````
+The official MongoDB Node.js driver lets you connect your Node.js
+applications to MongoDB and work with your data. The driver features an
+asynchronous API which allows you to access method return values through
+Promises or specify callbacks to access them when communicating with
+MongoDB.
 
-.. list-table::
-
-   * - .. cssclass:: bordered-figure
-       .. figure:: /includes/figures/M220JS_hero.jpg
-          :alt: M220JS course banner
-
-     - `M220JS: MongoDB for JavaScript Developers <https://university.mongodb.com/courses/M220JS/about>`_
-        Learn the essentials of Node.js application development with MongoDB.
+On this page, you can find descriptions of each section of the driver
+documentation and where to go to learn more about the Node.js driver.
 
 Quick Start
 -----------
-To get up and running quickly, take a look at the :doc:`Quick Start
-</quick-start>` to learn how to establish a connection to MongoDB Atlas
-and begin to work with data.
+
+The :doc:`Quick Start </quick-start>` section is where you can
+establish a connection to MongoDB Atlas and begin working with data.
 
 Usage Examples
 --------------
-The :doc:`Usage Examples </usage-examples>` section provides
-runnable code snippets and explanations for common methods. We recommend
-this section for users that are new to the MongoDB Node.js driver.
+
+The :doc:`Usage Examples </usage-examples>` section is where you can
+find runnable code snippets and high-level explanations for common
+methods.
 
 Fundamentals
 ------------
-Whether you are new to MongoDB or you just want to brush up on the core
-concepts, you can check out the **Fundamentals** section which features
-material on using the Node.js driver for the following:
+
+The Fundamentals section explains how to perform the following tasks
+using the Node.js driver:
 
 .. include:: /includes/fundamentals-sections.rst
 
 API
 ---
-See the `API documentation <{+api+}>`__ if you are looking for
-technical information about classes, methods, and configuration objects
-within the MongoDB Node.js driver.
+
+The `MongoDB Node.js driver API documentation <{+api+}>`__ contains
+detailed information about classes and methods in the MongoDB
+Node.js driver.
 
 FAQ
 ---
+
```

Comment:
I'd suggest rewording similarly for FAQ, Issues & Help, Compatibility, and What's New. Perhaps you could give it a try before I make suggestions?